Bluepill-cli/Bluepill-cli/Bluepill/Connections/BPTestBundleConnection.h

```diff
@@ -0,0 +1,28 @@
+//
+//  BPTestBundleConnection.h
+//  Bluepill-cli
+//
+//  Created by Keqiu Hu on 2/7/17.
+//  Copyright © 2017 LinkedIn. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "XCTestManager_IDEInterface-Protocol.h"
+#import "SimDevice.h"
+#import "BPConfiguration.h"
+#import "BPSimulator.h"
+
+// This is a small subset of XCTestManager_IDEInterface protocol
```

Comment:
By far, that is the only one we need to delegate out. We can potentially leverage others later